instructions/instruction.md

```diff
@@ -0,0 +1,21 @@
+![Clio logo](logo.png)
+
+Welcome to the Clio React test. The following test contains a small design that you can use for reference and multiple steps that you will illustrate what to do. You can work on each step in the order that you prefer.
+The test can be found at https://github.com/clio-me/react-test you can either download the folder or fork the project :)
```

Comment:
Given this page is linked within the README.md file, does it make sense to have this back-link?

Reply:
Thanks for feedback, I've removed the sentence